lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -137,6 +137,22 @@ def initialize(model, current_user:, embed_links: false)
           }
         end
 
+        link :'custom-fields' do
+          {
+            href: 'custom-fields',
+            type: 'text/html',
+            title: "Custom fields"
```

Comment:
If you use the translation as the title, you won't have to provide translations in the frontend.